watershed-android/watershed/app/src/main/AndroidManifest.xml

```diff
@@ -27,6 +30,9 @@
             android:screenOrientation="portrait">
         </activity>
         <activity android:name="com.facebook.LoginActivity"/>
+        <meta-data
+            android:name="com.crashlytics.ApiKey"
+            android:value="54afb894a34a5e0e989e1cedb236dc72cd0acd8d" />
```

Comment:
Probably not, but there doesn't seem to be a great solution:

http://stackoverflow.com/questions/14570989/best-practice-for-storing-private-api-keys-in-android
http://stackoverflow.com/questions/5525305/how-to-store-a-secret-api-key-in-an-applications-binary